gitbook-docs/streaming_api.md

```diff
@@ -23,3 +23,17 @@ Upon connection a 'hello' message is sent as follows:
   "self": "123456789"
 }
 ```
+
+#### Streaming: raw TCP sockets
+
+A signalk server MAY stream signalk-delta over a raw TCP socket.
+
+When signalk-delta is streamed over a raw TCP socket:
+
+1. The first character of each signalk-delta message MUST be the openning curly bracke (char(123), '{').
```

Comment:
typo: opening curly bracket

The wording here is a bit ambiguous on whether the JSON object is surrounded by {} or not. Suggest:

The stream MUST comprise a set of compact JSON objects separated by a carriage return, new line sequence (char(13) followed by char(10), "\r\n"). Each JSON object SHOULD conform to the JSON schema for a signalk-delta message.